### PR TITLE
add marker to mini map in event.html

### LINKED ISSUE #54: Add mini map or geolocation
allows users to place a marker on a map without using coordinates

### PR DESCRIPTION
**Pull request by oasemota99**
allows user to place marker on mini map in event.html using a single click (be sure NOT to double click)
fixes #54 
*** NOTE ***
This PR simply adds the functionality of creating a marker on a mini map. I don't recommend extensive testing of it. It certainly does not fully replace the current maps feature using coordinates.

### CODE REVIEW

File: capstone/src/main/resources/static/event.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ function createEventElement(eventListElement, event, displaySaveButton) {
 
   // Displays only for individual users
   if (displaySaveButton) {
-    eventElement.appendChild(createSaveEventButton(event));
```
Comment (elefeint) [inline capstone/src/main/resources/static/event.js:59]: Does this change belong in this PR? Looks like something that might have dropped in from another one.

File: capstone/src/main/resources/static/map.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     google.maps.event.addListener(campusMap, 'click', function(event) {
+      placeMarkerAndPan(event.latLng, campusMap);
+     });
+  }
+
+/* Create a new marker for each event
+ * @param event - event object
+ * @param campusMap - Google Map object
+ */
+function createMarker(event,campusMap) {
```
Comment (elefeint) [inline capstone/src/main/resources/static/map.js:34]: Really great naming and helper functions. Makes it easy to read the main logic.

File: capstone/src/main/resources/static/map.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+/* Function to create Google Map in map.html page */
+async function createMap() {
+  var princetonLatLng = {lat: 40.3428452, lng: -74.6568153};
+  const campusMap = new google.maps.Map(
+    document.getElementById('map'),
+    {center: princetonLatLng, zoom: 16});
+
+   google.maps.event.addListener(campusMap, 'click', function(event) {
```
Comment (meltsufin) [inline capstone/src/main/resources/static/map.js:8]: You probably don't need this functionality on the main events map.

File: capstone/src/main/resources/static/map.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+/* Function to create Mini Map in event.html page */
+  function createEventPlacementMap() {
+    var princetonLatLng = {lat: 40.3428452, lng: -74.6568153};
+    const campusMap = new google.maps.Map(
+      document.getElementById('eventMap'),
+      {center: princetonLatLng, zoom: 16});
+
+     google.maps.event.addListener(campusMap, 'click', function(event) {
+      placeMarkerAndPan(event.latLng, campusMap);
```
Comment (meltsufin) [inline capstone/src/main/resources/static/map.js:26]: I guess you're still working on populating the lat and long in the event form based on the click, which would go right after this line.

File: capstone/src/main/resources/static/event.js (hunk 1 above)
Comment (oasemota99) [reply capstone/src/main/resources/static/event.js]: I have no idea where this came from. I'm reverting the change

File: capstone/src/main/resources/static/map.js (hunk 4 above)
Comment (oasemota99) [reply capstone/src/main/resources/static/map.js]: resolved!

**Pull request merged**